README.mkd

```diff
@@ -30,34 +30,36 @@ seek input from the wider GCC and Clang developer communities for extensions
 or changes beyond the current set of command-line options supported by GCC.
 
 See the [issues 
-list](https://github.com/lowRISC/riscv-toolchain-conventions/issues) to 
+list](https://github.com/riscv-non-isa/riscv-toolchain-conventions/issues) to 
 discuss any of the problems or TODO items described in this document.
 
 This document is currently targeted at toolchain implementers and developers, 
 but over time we hope it will also become a useful reference for RISC-V 
 toolchain users.
 
 ## See also
-* [RISC-V user-level ISA specification](https://riscv.org/specifications/)
+* [RISC-V user-level ISA specification](https://riscv.org/technical/specifications/)
+  (Document Version 20191213)
 * [RISC-V ELF psABI 
-specification](https://github.com/riscv/riscv-elf-psabi-doc/blob/master/riscv-elf.md)
+specification](https://github.com/riscv-non-isa/riscv-elf-psabi-doc/blob/master/riscv-elf.adoc)
 * [RISC-V Assembly Programmer's 
-Manual](https://github.com/riscv/riscv-asm-manual/blob/master/riscv-asm.md)
+Manual](https://github.com/riscv-non-isa/riscv-asm-manual/blob/master/riscv-asm.md)
 * [GCC RISC-V option 
 documentation](https://gcc.gnu.org/onlinedocs/gcc/RISC-V-Options.html)
 
 ## Specifying the target ISA with -march
 
 The compiler and assembler both accept the `-march` flag to specify the target 
-ISA, e.g. "rv32imafd". The abbreviation "g" can be used to represent the IMAFD 
-base and extensions, e.g. `-march=rv64g`. A target `-march` which includes 
-floating point instructions implies a hardfloat calling convention, but can be 
-overridden using the `-mabi` flag (see the next section).
-
-The ISA subset naming conventions are described in Chapter 22 of the RISC-V 
+ISA, e.g. "rv32imafd". The abbreviation "g" can be used to represent either 
+`IMAFD` (when targeting RISC-V ISA specification version 2.2 or before) or 
```

Comment:
nit: "or before" -> "or earlier"?

Reply:
That would work. I'm not a native speaker so this kind of feedback is always helpful.

Reply:
I reflected your comment in the document (before → earlier). Thanks!